converter/codec.go

```diff
@@ -210,7 +214,7 @@ func (e *CodecDataConverter) ToPayloads(value ...interface{}) (*commonpb.Payload
 		}
 	}
 
-	if len(rawValuePayloads) > 0 {
```

Comment:
It is a bit confusing to only handle raw values here is all of the inputs are raw value. Maybe we should just rely on the underlying data converter? Originally I was hopping we could make `CodecDataConverter` support raw value, but it looks like that may not be possible to handle it consistently. What do you think?